icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2777,7 +2781,9 @@ class U_I18N_API FormattedNumber : public UMemory, public FormattedValue {
 };
 
 #ifndef U_HIDE_DRAFT_API
-// Note: This is draft ICU 65
+/**
+ * @draft ICU 65
+ */
 template<typename StringClass>
```

Comment:
Draft status was not marked properly. There were some other places like this that I also fixed.

Reply:
Actually, these are the inline impls of methods that are now `@stable`.  They aren't docstrings.  We should just remove the conditional.

Reply:
Thanks, good catch. But this one was the only one that was now stable. I removed the conditionals for this one and changed the other two places so the comment clarifies that it is the inline impl of a @draft ICU 68 method.